scripts/gen_iptables.sh

```diff
@@ -13,52 +13,53 @@ source "$PROJECT_DIR/scripts/parse_site_config.sh"
 parse_site_config "$PROJECT_DIR/cluster/site-config.yaml" "./cluster" || exit 1
 map_site_config "true" || exit 1
 
-ins_del_rule()
-{
-    operation=$1
-    table=$2
-    rule=$3
-   
-    if [ "$operation" == "INSERT" ]; then
-        if ! sudo iptables -t "$table" -C $rule > /dev/null 2>&1; then
-            sudo iptables -t "$table" -I $rule
-        fi
-    elif [ "$operation" == "DELETE" ]; then
-        sudo iptables -t "$table" -D $rule
-    else
-        echo "${FUNCNAME[0]}: Invalid operation: $operation"
-        exit 1
-    fi
-}
-
 # allow DNS/DHCP traffic to dnsmasq and coredns
-ins_del_rule "INSERT" "filter" "INPUT -i $BM_BRIDGE -p udp -m udp --dport 67 -j ACCEPT"
-ins_del_rule "INSERT" "filter" "INPUT -i $BM_BRIDGE -p udp -m udp --dport 53 -j ACCEPT"
-ins_del_rule "INSERT" "filter" "INPUT -i $BM_BRIDGE -p tcp -m tcp --dport 67 -j ACCEPT"
-ins_del_rule "INSERT" "filter" "INPUT -i $BM_BRIDGE -p tcp -m tcp --dport 53 -j ACCEPT"
-ins_del_rule "INSERT" "filter" "INPUT -i $BM_BRIDGE -p tcp -m tcp --dport 6443 -j ACCEPT"
-ins_del_rule "INSERT" "filter" "INPUT -p udp -i $PROV_BRIDGE --dport 67 -j ACCEPT"
-ins_del_rule "INSERT" "filter" "INPUT -p udp -i $PROV_BRIDGE --dport 69 -j ACCEPT"
-ins_del_rule "INSERT" "filter" "INPUT -p tcp -i $PROV_BRIDGE --dport 8080 -j ACCEPT"
+sudo systemctl start firewalld
+sudo firewall-cmd --add-interface="$BM_BRIDGE" --zone=public --permanent
+sudo firewall-cmd --add-interface="$PROV_BRIDGE" --zone=public --permanent
+sudo firewall-cmd --zone=public --permanent --add-service=ssh
+sudo firewall-cmd --add-service=http --zone=public --permanent
+sudo firewall-cmd --add-service=https --zone=public --permanent
+sudo firewall-cmd --zone=public --permanent --add-port=67/udp
+sudo firewall-cmd --zone=public --permanent --add-port=53/udp
+sudo firewall-cmd --zone=public --permanent --add-port=67/tcp
+sudo firewall-cmd --zone=public --permanent --add-port=53/tcp
+sudo firewall-cmd --zone=public --permanent --add-port=6443/tcp
+sudo firewall-cmd --zone=public --permanent --add-port=67/udp
+sudo firewall-cmd --zone=public --permanent --add-port=69/udp
+sudo firewall-cmd --zone=public --permanent --add-port=8080/tcp
+sudo firewall-cmd --zone=public --permanent --add-port=22623/tcp
+
+#Networking requirements for user-provisioned infrastructure
```

Comment:
this is just needed when we do a virtualized install, but well at this time we don't know anything about the type of cluster that we are going to deploy. Andrew, what do you think, do we always set the rules? or we do some conditional depending on the type of cluster to install?

Reply:
I'd say just include them for now unless they're somehow causing an issue.